editor/app/(www)/(figma)/figma/ci/page.tsx

```
@@ -0,0 +1,404 @@
+                <span className="text-teal-500">&quot;./grida/home&quot;</span>
+                <span>;</span>
+              </span>
+            );
+          }
+        }
+
+        // Return other lines unchanged
+        return <span key={lineIndex}>{line}</span>;
+      })
+      .reduce((acc: React.ReactNode[], element, index) => {
+        return [...acc, element, <br key={`br-${index}`} />];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)